R/print.R

```diff
@@ -442,6 +442,66 @@ print.bbi_nmboot_summary <- function(x, .digits = 3, .nrow = 10, ...) {
   }
 }
 
+
+#' @describeIn print_bbi Prints the `NM-TRAN` evaluation of a `bbi_nonmem_model`
+#'  object
+#' @export
+print.nmtran_process <- function(x, ...){
+
+  is_valid_print <- function(.x) {
+    if (!is.null(.x)) {
+      length(.x) != 0
+    } else {
+      FALSE
+    }
+  }
+
+  heading <- cli_h1
+  subheading <- cli_h2
+  bullet_list <- cat_bullet
+
+  status <- x[["status"]]
+  if (x[["status_val"]] == 0) {
+    status <- col_green(status)
+  } else {
+    status <- col_red(status)
+  }
+
+  nm_version <- x[["nonmem_version"]]
+  if(is.null(nm_version)) nm_version <- "unknown"
+
+  heading('Status')
+  subheading(status)
+
+  heading("Absolute Model Path")
+  bullet_list(x[[ABS_MOD_PATH]])
+
+  heading("NMTRAN Specifications")
+  cli::cli_bullets(
+    c(
+      "*" = paste0("NMTRAN Executable:  {.path ", x[["nmtran_exe"]], "}"),
+      "*" = paste0("NONMEM Version: {.val ", nm_version, "}"),
+      "*" = paste0("Run Directory: {.path ", x[["run_dir"]], "}")
+    )
+  )
```

Comment:
This could be reworked to remove the `paste0` calls:

```R
cli::cli_bullets(
  c(
    "*" = "NMTRAN Executable: {.path {x[['nmtran_exe']]}}",
    "*" = "NONMEM Version: {.val {nm_version}}",
    "*" = "Run Directory: {.path {x[['run_dir']]}}"
  )
)
```